Mathlib/Analysis/NormedSpace/FiniteDimension.lean

```diff
@@ -231,6 +231,32 @@ theorem LinearMap.exists_antilipschitzWith [FiniteDimensional 𝕜 E] (f : E →
     exact ⟨_, e.nnnorm_symm_pos, e.antilipschitz⟩
 #align linear_map.exists_antilipschitz_with LinearMap.exists_antilipschitzWith
 
+open Function in
+/-- A `LinearMap` on a finite-dimensional space over a complete field
+  is injective iff it is anti-Lipschitz. -/
+theorem LinearMap.injective_iff_antilipschitz
+    [FiniteDimensional 𝕜 E] [CompleteSpace 𝕜] (f : E →ₗ[𝕜] F) :
+    Injective f ↔ ∃ K > 0, AntilipschitzWith K f := by
+  constructor
+  · rw [← LinearMap.ker_eq_bot]
+    exact f.exists_antilipschitzWith
+  · rintro ⟨K, -, H⟩
+    exact H.injective
+
+open Function in
+/-- Injectivity of continuous linear maps `E → F` is open,
```

Comment:
This phrasing to me sounds a bit better.
```suggestion
/-- The set of injective continuous linear maps `E → F` is open,
```

Reply:
I agree! "An open condition" is common, but I don't see a need to this jargon.